pkg/config/config.go

```diff
@@ -692,6 +692,7 @@ func InitConfig(config Config) {
 	config.SetKnown("system_probe_config.enable_conntrack")
 	config.SetKnown("system_probe_config.sysprobe_socket")
 	config.SetKnown("system_probe_config.conntrack_rate_limit")
+	config.SetKnown("system_probe_config.enable_conntrack_all_namespaces")
```

Comment:
Can you please make sure this can be also set through an ENV var?
(I can't remember it config `config.SetKnown` does that automatically

Reply:
I don't think it is automatically bound to a env var. None of the other system_probe config flags are, why this one?

Reply:
A few of them are. See https://github.com/DataDog/datadog-agent/blob/554d6eac3253316e2db30382ae4fa5d2641d6c11/pkg/process/config/config.go#L481-L499
That will make it easier to enable/disable this in containerized environments.

Reply:
Ok, let me add it.

Reply:
Done